spec/controllers/users/totp_setup_controller_spec.rb

```diff
@@ -32,10 +30,6 @@
           subject.current_user.decorate.qrcode(subject.user_session[:new_totp_secret])
         ).not_to be_nil
       end
-
-      it 'presents a QR code to the user' do
```

Comment:
Oh yeah, we can't really test that you can scan the qr code with your device in a feature spec. That's probably why we opted for rendering the view in the controller spec. I would think the controller spec is faster, even when rendering views, so maybe keep it here for now?